.cirrus.yml

```diff
@@ -0,0 +1,17 @@
+env:
```

Comment:
Either Cirrus-CI or Circle-CI offer arm64 linux runners for open-source/free use